boost.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "boost",
+    "buildsystem": "simple",
+    "build-commands": [
+        "./bootstrap.sh --prefix=/app",
```

Comment:
Do you need entire boost or just few particular libraries?

Reply:
Well, different libraries from boost are used by different components (Mapnik, Valhalla are the main offenders). I maybe able to track down what's exactly needed, but would prefer not to spend too much time on that at present.